cmake/config.h.in

```diff
@@ -0,0 +1,10 @@
+#ifndef GNURADIO_GRAPH_PROJECT_CONFIG
+#define GNURADIO_GRAPH_PROJECT_CONFIG
+
+#define CXX_COMPILER_PATH "@CMAKE_CXX_COMPILER@"
+#define CXX_COMPILER_ARG1 "@CMAKE_CXX_COMPILER_ARG1@"
+#define CXX_COMPILER_ID "@CMAKE_CXX_COMPILER_ID@"
+#define CXX_COMPILER_VERSION "@CMAKE_CXX_COMPILER_VERSION@"
+#define CXX_COMPILER_FLAGS "@ALL_COMPILER_FLAGS@"
```

Comment:
Very much in favor of introducing cmake config file.

Can you also add FMT availability to the config.h.in so that we can replace the problematic `ifdef FMT_FORMAT_H_` in our headers.